### DISCUSSION TITLE
My Bootcamp Webpage

### DISCUSSION THREAD

**Discussion by Andrewjr98** [community/community#28518] [Other Feature Feedback, Questions, & Ideas]
So I have made a website for my bootcamp notes that I need to have turned in. Unfortunately my README.md comes up instead of my index.html file.  When I try to make a new file and add said new file I have a JavaScript issue. So some help would be appreciated.

**Comment by radar-blip** [comment]
Try adding your index.html to the main repo and see what happens. 
I think you have some files mixed up is all.

**Reply by Andrewjr98** [reply]
So that works I just have to add index.html to the end. The other problem I’m having is my CSS and JavaScript files are no longer connected

**Reply by radar-blip** [reply]
Its just a matter of getting all your files in order, and links pointing to the right page.

**Reply by radar-blip** [reply]
That picture, is that batman or catwoman?

**Reply by Andrewjr98** [reply]
Haha I don’t think either. I think it’s just a drawing cat with a bow tie

**Reply by radar-blip** [reply]
I think it would work now if you put your .css and .js files into the main repo (with your index.html).
Then you would have no need for that extra folder (but keep it just in case).

But at the same time, there has to be a way to make things work with everything in the folder. Its like an either or type thing.
Im getting confused myself. lol

**Comment by jfullstackdev** [comment]
it's because of the URLs

![image](https://user-images.githubusercontent.com/47092464/183234648-096af79d-4184-4b3a-ad67-47d3e461f9da.png)

for example

```
<link rel="stylesheet" href="./assets/style.css">
```

you have two `assets` folder, that's why you are confused but given that current location, this will be the correct example

<https://andrewjr98.github.io/Pre-work-Study-guide/prework-study-guide/assets/style.css>

![image](https://user-images.githubusercontent.com/47092464/183234755-9e41de3b-18de-4f45-ad96-563725dda5ca.png)

and JavaScript

<https://andrewjr98.github.io/Pre-work-Study-guide/prework-study-guide/assets/script.js>

update those in your `index.html`
